Reference/V9-Config/HealthChecks/index-v9.md

```diff
@@ -0,0 +1,98 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Umbraco Health Check Settings"
+meta.Description: "Information on the health check settings section"
+state: complete
+verified-against: beta-3
+update-links: true
+---
+
+# Health checks
+
+The health checks section allows you to disable certain health checks, and configure your own custom notification methods, that will automatically run the health checks every so often, and notify you if any health checks fails.
+
+An example of a HealthChecks settings can look something like this:
+
+```json
+"Umbraco": {
+  "CMS": {
+    "HealthChecks": {
+      "DisabledChecks": [
+        {
+          "Id": "D0F7599E-9B2A-4D9E-9883-81C7EDC5616F"
+        }
+      ],
+      "Notification": {
+        "Enabled": true,
+        "FirstRunTime": "5 * * * *",
+        "Period": "1.00:00:00",
+        "NotificationMethods": {
+          "email": {
+            "Enabled": true,
+            "Verbosity": "Detailed",
+            "FailureOnly": true,
+            "Settings": {
+              "RecipientEmail": "alerts@mywebsite.tld"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+This config will disable the macro errors check, and enable notifications to run the check and notify via email if a check fails, running the first time five minutes are the site as booted, and this once every day.
```

Comment:
Not sure I understand the last part of this sentence 🤔

Reply:
I can see why hehe, I managed to butcher that sentence, I've tried updating it 😄